Core/Core/Analytics/Analytics.swift

```diff
@@ -27,6 +28,19 @@ public class Analytics: NSObject {
     @objc public static var shared: Analytics = Analytics()
     public weak var handler: AnalyticsHandler?
 
+    @objc
+    public func initialize() {
+        guard !ProcessInfo.isUITest, let heapID = getHeapID() else { return }
+        let options = HeapOptions()
+        options.disableAdvertiserIdCapture = true
+        Heap.initialize(heapID, with: options)
+    }
+
+    private func getHeapID() -> String? {
+        AppEnvironment.shared.isDebug ?
```

Comment:
Since the heap ID gets injected based on the environment (prod or dev) does it make sense to have a separate staging and production id? One ID would be enough I think, also this if debug logic could be removed.